src/handler/events/eventsHandler.ts

```diff
@@ -0,0 +1,11 @@
+import type Wrapper from '../structures/wrapper';
+import type { Observable } from 'rxjs';
+import { Subject } from 'rxjs';
```

Comment:
You prefer this (1. current) or
2.
```ts
import { Subject, type Observable } from 'rxjs';
```
or
3.
```ts
import { Observable, Subject } from 'rxjs';
```

Reply:
Definitely two, i didnt know u could do that

Reply:
xD